Code/Framework/AzCore/AzCore/Memory/Memory.h

```diff
@@ -734,12 +734,15 @@ namespace AZ
         public:
             typedef typename Allocator::Descriptor Descriptor;
 
-            AZ_FORCE_INLINE static IAllocatorAllocate& Get()
+            // Maintained for backwards compatibility, prefer to use Get() instead.
+            // Get was previously used to get the the schema, however, that bypases what the allocators are doing.
```

Comment:
nitpick: typo `bypases` -> `bypasses`
```suggestion
            // Get was previously used to get the the schema, however, that bypasses what the allocators are doing.
```